src/main/com/cqfn/degitx/simulator/ElectorState.kt

```diff
@@ -0,0 +1,9 @@
+package com.cqfn.degitx.simulator
+
+/**
+ * State of node in Election process.
+ * @see GitRepo
+ */
+enum class ElectorState {
```

Comment:
@IlyaMoskva as we discussed earlier, we don't need to implement election process of nodes on simulator level, it will be the part of simulating software algorithm.

Reply:
@g4s8 , right, it is not needed. It was old design.